### ISSUE TITLE
Editing the text in Post Author Name block

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/gutenberg#40157]
## What problem does this address?

What if I wanted to add to the Post Author name block. 
Such as All posts by Joachim, About Joachim, Written by Joachim, Written by a Norwegian, core contributor PJ etc etc.
Currently we are not able to adjust the text contents. 

![Screenshot 2022-04-07 at 16 41 20](https://user-images.githubusercontent.com/5323259/162225637-1664d4b1-4b8e-4ed4-a0c9-6db458d66c2d.jpg)

## What is your proposed solution?

Being able to adjust the text contents of the Post Author Name. Calling the author by another nick name, adding other text etc. 

@c4rl0sbr4v0 @carolinan

**Comment by carolinan** [comment]
You can use any other text block like a heading or paragraph block and place it before the post author name.
If we would keep the texts in the same block, then all settings would apply to all the text so you wouldn't be able to have separate font sizes, font families etc.

**Comment by paaljoachim** [comment]
I just tried.... then suddenly I figured out how to work with the Row feature. I was nicely surprised.

https://user-images.githubusercontent.com/5323259/162232473-4f19e5b2-7dd8-4ff0-b87b-b1c86074e8e7.mp4

**Comment by mvaleno** [comment]
> You can use any other text block like a heading or paragraph block and place it before the post author name. If we would keep the texts in the same block, then all settings would apply to all the text so you wouldn't be able to have separate font sizes, font families etc.

I don't have the "Post Author Name" block, only a "Post Author" Block which does not have the option to "link to author archive" ? 



Why is this not showing up in my core blocks? Do you need to enable some of the core blocks?

**Comment by cbravobernal** [comment]
Hi @mvaleno
I guess it could be because you need to have Guteberg plugin installed to see those blocks.

**Comment by mvaleno** [comment]
Thank you. So, the Gutenberg plugin code is ahead of the code in WP Core? Is that how it works? Do I understand this correctly?

Meaning, features are added to Gutenberg plugin, then later baked into WP Core?

**Comment by cbravobernal** [comment]
Yes, kind of, it is early access to the Core editor. That means that it can fastly change before baking into the Core, ie, sometimes the community decides not to include some features due to many reasons (backward compatibility, not enough traction or testing, impact on the WordPress ecosystem, etc.)
